apps/src/templates/instructions/InstructionsCsfLeftCol.jsx

```diff
@@ -51,12 +53,34 @@ class InstructionsCsfLeftCol extends React.Component {
    * Handle a click to the hint display lightbulb
    */
   handleClickLightbulb = () => {
-    // If we don't have authored hints to display, clicking lightbulb shouldnt do anything
+    // If we don't have authored hints to display, clicking lightbulb shouldn't do anything.
     if (this.props.hasAuthoredHints && this.props.hasUnseenHint) {
       this.props.requestHint();
     }
   };
 
+  /**
+   * Handle an enter key press to the hint display lightbulb
+   */
+  handleKeyDownLightbulb = e => {
+    // If we don't have authored hints to display, keyboard-navigating to lightbulb
+    // and then pressing 'enter' shouldn't do anything.
+    if (
```

Comment:
Is it undesirable to have this behavior if you click on it?

Reply:
Good question - I don't think so and I think it makes sense to have similar behavior. Thanks! will update.